### PR TITLE
👽 [CLW3] Update for new API

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/server/api/routes/civic/staking.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+  apiCivicLikerGetStaking,
+  apiCivicLikerGetStakingInfo,
+} = require('../../util/api');
 
 const router = Router();
 
+router.get('/civic/staking/info', async (req, res, next) => {
+  try {
+    const { data } = await apiCivicLikerGetStakingInfo(req);
+    res.set('Cache-Control', 'public, max-age=86400');
```
Comment (williamchong) [inline src/server/api/routes/civic/staking.js:13]: please add stake while revalidate

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/server/api/routes/civic/staking.js (hunk 1 above)
Comment (nwingt) [reply src/server/api/routes/civic/staking.js:13]: What do you mean?

Comment (williamchong) [reply src/server/api/routes/civic/staking.js:13]: stale while revalidate, stupid auto correct...

Comment (williamchong) [reply src/server/api/routes/civic/staking.js:13]: e.g. `Cache-Control: max-age= 86400, stale-while-revalidate=86400`

Comment (nwingt) [reply src/server/api/routes/civic/staking.js]: Updated

**Pull request merged**